templates/space/index.html.twig

```diff
@@ -32,7 +32,13 @@
     </div>
 {{ form_end(form) }}
 {% for space in spaces %}
-
-<p>{{ space.name }}</p>
+<article class="offer">
+    <img class="image-offer" src="{{ asset('build/images/imageHome.svg') }}" height="160" alt="Marché Conclu"/>
+    <div class="text-offer">
+        <p>{{ space.name }}</p>
+        <p>{{ space.category }}</p>
+        <p>{{ space.surface }}</p>
+    </div>
+</article>
```

Comment:
an img with additional content... looks like a figure more than an article, doesn't it? and so the div should be a figcaption

Reply:
Done !